server/src/main/java/org/apache/cassandra/sidecar/config/yaml/CdcConfigurationImpl.java

```diff
@@ -32,33 +33,58 @@
 public class CdcConfigurationImpl implements CdcConfiguration
 {
     private static final Logger LOGGER = LoggerFactory.getLogger(CdcConfigurationImpl.class);
+    public static final String IS_ENABLED_PROPERTY = "is_enabled";
```

Comment:
to maintain consistency with the existing configuration this should just be "enabled"
```suggestion
    public static final String IS_ENABLED_PROPERTY = "enabled";
```